ios/ImagePickerManager.m

```diff
@@ -201,13 +202,19 @@ -(NSMutableDictionary *)mapVideoToAsset:(NSURL *)url error:(NSError **)error {
           }
         }
     }
-    
+
     NSMutableDictionary *asset = [[NSMutableDictionary alloc] init];
     asset[@"fileName"] = fileName;
     asset[@"duration"] = [NSNumber numberWithDouble:CMTimeGetSeconds([AVAsset assetWithURL:videoDestinationURL].duration)];
     asset[@"uri"] = videoDestinationURL.absoluteString;
     asset[@"type"] = [ImagePickerUtils getFileTypeFromUrl:videoDestinationURL];
     asset[@"fileSize"] = [ImagePickerUtils getFileSizeFromUrl:videoDestinationURL];
+
+    if (phAsset) {
+      asset[@"id"] = phAsset.localIdentifier;
+      // Add more extra data here ...
+    }
```

Comment:
Tomorrow I'll be adding video exif data here, as we need it in our product, this is a great update! But I wonder in the future if we could use some abstraction for `mapVideoToAsset` and `mapImageToAsset`, not suggesting a change here, just want to hear your thoughts

Reply:
yeah I think that could be good!! Will reduce duplication and make it clearer which props are shared across videos and images